plugins/basic/README.md

```diff
@@ -0,0 +1 @@
+every plugin should have a readme file with info.
```

Comment:
Let's add build instructions to this file so that it's easy for a developer to undertstand how to get started with plugins.